valdr-ng/projects/valdr-ng-showcase/tsconfig.app.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "outDir": "../../out-tsc/app",
     "resolveJsonModule": true,
+    "strictPropertyInitialization": false,
```

Comment:
That's to avoid having all types with a | null, right?

Reply:
Exactly. To be precise, it was the input for `Person`, and this only affects the showcase.